tlslite/utils/asn1parser.py

```diff
@@ -90,3 +118,23 @@ def _getASN1Length(p):
         else:
             lengthLength = firstLength & 0x7F
             return p.get(lengthLength)
+
+    @staticmethod
+    def _getASN1Type(p):
```

Comment:
nit: `p` is a bit short variable name, `parser` probably would be better
`_getASN1Type()` is camelCase style, `_get_asn1_type()` (or even `_parse_type()`) would be more pythonic